src/main/java/uk/gov/hmcts/reform/sscs/service/servicebus/TopicPublisher.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.sscs.service.servicebus;
+
+import java.util.concurrent.atomic.AtomicReference;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.Session;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jms.IllegalStateException;
+import org.springframework.jms.connection.CachingConnectionFactory;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class TopicPublisher {
+
+    private final JmsTemplate jmsTemplate;
+
+    private final String destination;
+
+    private final ConnectionFactory connectionFactory;
+
+    @Autowired
+    public TopicPublisher(JmsTemplate jmsTemplate,
```

Comment:
Where is this JmsTemplate bean defined? looks like we'll need MessagingConfig too